help/C/Help_ch_Transactions.xml

```diff
@@ -891,6 +891,12 @@ Author:
           </para></listitem>
           <listitem><para><emphasis role="bold"><link linkend="trans-import-csv">CSV</link></emphasis> 
                (.csv)  Comma Separated Values;</para></listitem>
+          <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">MT940</link></emphasis> 
+               (.STA)  SWIFT MT940 Customer Statement Message;</para></listitem>
```

Comment:
1. While the 3 first formats can hold transactions for several accounts in one file, this 3 usually hold transactions for one bank account (or CC).
1.  From where do you have this and the next 2 filename extensions? AFAIK, there are none defined.
1. The official description is "end-of-day bank account statement". [https://en.wikipedia.org/wiki/MT940] In contrast 942 is an intra-day statement.

Reply:
Frank, 
I can add a note to the effect that the MT940 is an end of day and MT942 is an intra day statement. The .STA extension came form a Polish bank's description of the MT940 format but it is probably just their practice. I also used http://www.societegenerale.rs/fileadmin/template/main/pdf/SGS%20MT940.pdf when I was writing it for info. Found it very difficult to locate official SWIFT documentation. noreference to MT940 or MT942 exists on their website possible because ISO 20022 has superseded them for interbank use. I presume the ISO 20022 is a front end to encrypted link for the exchange. 

If the use of a specific extension is not mandatory or even  common I will drop it . I lumped the three formats together primarily because there is little setup involved in importing them apart from loading the specific  files as the internal format of the file is quite strict. I had a couple of sample DTAUS  MT940 and MT942 files to play with that I downloaded from a couple of European bank sites.